src/js/components/Grid/doc.js

```diff
@@ -128,8 +131,11 @@ space in the row axis.`,
     rows: PropTypes.oneOfType([
       PropTypes.arrayOf(
         PropTypes.oneOfType([
+          PropTypes.arrayOf(PropTypes.OneOfType([
```

Comment:
This seems unrelated with your PR. can you explain what you are doing here? The ts definitions don't need an update?

Reply:
@oorestisime this one happened here b/c there is another unrelated bug. The prop topes for column and down is self-inconsistent. I didn't have time to file a separate issue for this and and this fix on my branch. Sorry for the confusion!!

The issue this is trying to address is the following: PropTypes of column (and row) in the past looked like 

```javascript
"xsmall"
"small"
"medium"
"large"
"xlarge"
"any CSS size"
["small", "..."]
[["small", "medium"], "..."]
```
*and* "string". "string" type is like an escape for thins that are not in the `OneOf` list.

However, the list of tuple for the lower and  higher range did not allow one to use `string` type. this is self-inconsistent. Which is why this line changes it so that the inner tuples in the ranged layout can also take non-`OneOf` strings.